elytron-oidc-client/src/main/resources/schema/wildfly-elytron-oidc-client_3_0.xsd

```diff
@@ -1189,6 +1189,16 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
+            <xs:element name="scope" type="xs:string" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        <![CDATA[
+                        The scope value are used to specify the use of OpedID Connect when authenticating.
+                        Additional scope values can be configured here.
```

Comment:
Could reword to something like:

A space separated list of scope values that should be used in the OIDC authentication request. Scopes can be used to request that specific sets of information be made available as Claim Values.